src/test/scala/scalismo/io/MeshIOTests.scala

```diff
@@ -154,4 +177,30 @@ class MeshIOTests extends ScalismoTestSuite {
 
   }
 
+  def createRandomTetrahedralMesh(): TetrahedralMesh3D = {
+    // points around unit cube
+
+    val rng = Random(42l)
+    val N = 200
+    val points = IndexedSeq.fill(N)(Point(
+      rng.scalaRandom.nextGaussian() * 2,
+      rng.scalaRandom.nextGaussian() * 1000,
+      rng.scalaRandom.nextGaussian() * 1000000
+    ))
+    val domain = UnstructuredPointsDomain(points)
+
+    // cells covering the complete cube
+    implicit def intToPointId(i: Int): PointId = PointId(i)
+    val T = 2
```

Comment:
why only 2 cells (T=2) from 200 points (N=200)?
For tetrahedral mesh, the number of cells is larger than the number of points

Reply:
In principle 2 is enough, while it actually is a change for a quick test that accidentally made it here. The mesh should not be anyhow reasonable, more the other way round, it should be special so that possible errors in the IO methods would show up quickly.

I set both to 200.